chart/slackconnector/templates/application.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: applicationconnector.kyma-project.io/v1alpha1
+kind: Application
+metadata:
+  name: {{ .Release.Name }}-app
+  labels:
+    app: {{ include "slackconnector.name" . }}
+    chart: {{ include "slackconnector.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    app.kubernetes.io/name: {{ template "slackconnector.name" . }}
+    helm.sh/chart: {{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}
+    app.kubernetes.io/managed-by: {{ .Release.Service }}
+    app.kubernetes.io/instance: {{ .Release.Name }}
+
+
```

Comment:
I think you should consider complying with [this](https://helm.sh/docs/chart_best_practices/#whitespace-in-generated-templates) Helm Best Practices suggestion in all your files. 

I'm not also sure if your labels are proper - they seem duplicated, but ask someone if that's right.

Reply:
Removed the whitespaces, also not sure about the labels